packages/@lwc/engine/src/framework/decorators/wire.ts

```diff
@@ -32,7 +35,7 @@ function wireDecorator(
             );
         }
     }
-    return createTrackedPropertyDescriptor(
+    return createWiredPropertyDescriptor(
```

Comment:
instead of using the tracked field for wired fields, we now have a different implementation.